feat: Update content to align with current marketing messaging (#47)
qctrl/.github: profile/README.md

### Who we are

We are a product-led, quantum control infrastructure software company solving real world problems, and we abide by [these standards](https://code.q-ctrl.com/).

[**Meet the team**](https://github.com/orgs/qctrl/people)

---

### What we do

> Products for everyone: Whether you are a newcomer, an enterprise user, or a hardware team, we make quantum technology useful for you - all through software.

Reviewer: https://learn.microsoft.com/en-us/style-guide/punctuation/dashes-hyphens/emes
```suggestion
Products for everyone: Whether you are a newcomer, an enterprise user, or a hardware team, we make quantum technology useful for you—all through software.
```

- [**Black Opal**](https://q-ctrl.com/black-opal): Start learning quantum computing.

  Upskill for the quantum era and empower your team with the best edtech solution for individuals and enterprises.

- [**Fire Opal**](https://q-ctrl.com/fire-opal): Find true value from quantum algorithms.

>   Gain >1,000X performance improvement on real hardware and reduce compute costs >100X with automated error suppression.

Reviewer: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/multiplication-sign
```suggestion
  Gain >1,000× performance improvement on real hardware and reduce compute costs >100X with automated error suppression.
```

- [**Boulder Opal**](https://q-ctrl.com/boulder-opal): Build the quantum platforms of the future.

>   The world’s first and most powerful design-and-test EDA for the quantum era.

Reviewer: Replacing a curly quote with a straight one
```suggestion
  The world's first and most powerful design-and-test EDA for the quantum era.
```
